message_adapter/message_adapter.py

```diff
@@ -109,14 +109,12 @@ def loadAndUpdateRemoteEvent(self, event, context):
         * @returns {*} the full event data
         """
         if 'replace' in event:
-            exception = 'None'
-            if 'exception' in event:
-                exception = event['exception']
+            exception = event.get('exception', None)
```

Comment:
Should it be 'None'?

Reply:
No, because this way we can take advantage of `not exception`.